### PR TITLE
Allow users to specify credentials in ENVARS

### PR DESCRIPTION
**Pull request by zacharyestep**
Sharan has requested you review this

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -248,6 +248,17 @@ The possible options for each credential profile are:
 Future versions of cbapi will also provide the ability to "pin" the TLS certificate so as to provide certificate
 verification on self-signed or internal CA signed certificates.
 
+Environment Variable Support
```
Comment (jgarman) [inline docs/index.rst]: this needs to explicitly state the hierarchy - Environment variables take precedence over a file

File: docs/index.rst [hunk 2]
```diff
@@ -248,6 +248,17 @@ The possible options for each credential profile are:
...
+
+The latest cbapi for python supports specifying API credentials in the following three environment variables:
+
+`CBAPI-TOKEN` the envar for holding the CbR/CbP api token or the ConnectorId/APIKEY combination for Cb Defense/PSC.
```
Comment (jgarman) [inline docs/index.rst]: this should be `CBAPI_TOKEN` (note underscore)

File: docs/index.rst [hunk 3]
```diff
@@ -248,6 +248,17 @@ The possible options for each credential profile are:
...
+
+The `CBAPI_URL` envar holds the FQDN of the target, a CbR , CBD, or CbD/PSC server specified just as they are in the
+configuration file format specified above.
+
+The `CBAPI_SSL_VERIFY` envar can be used to control SSL validation, which will default to ON.
```
Comment (jgarman) [inline docs/index.rst]: `ON` isn't a valid value for this - it's `true` or `false`? in fact... the code below doesn't support this very well

File: src/cbapi/auth.py [hunk 4]
```diff
@@ -60,7 +65,7 @@ def __init__(self):
         environ = os.environ
         self.cbapi_url = environ.get('CBAPI_URL',None)
         self.cbapi_token = environ.get('CBAPI_TOKEN',None)
-        self.cbapi_ssl_verify = environ.get('CBAPI_SSL_VERIFY',None)
+        self.cbapi_ssl_verify = environ.get('CBAPI_SSL_VERIFY',True)
```
Comment (jgarman) [inline src/cbapi/auth.py]: This doesn't work the same as the file credential store - see https://github.com/carbonblack/cbapi-python/blob/master/src/cbapi/auth.py#L46

Comment (jgarman) [reply src/cbapi/auth.py]: basically this code won't work as written - if you put "FALSE" or anything in `CBAPI_SSL_VERIFY` it will be "true"

**Comment by jgarman** [comment]
I need you to address those issues above ^^ sorry I didn't see the issue with `CBAPI_SSL_VERIFY` before but these need to be addressed before we can release

**Comment by zacharyestep** [comment]
I will update the index.rst to reflect the above. There is no issue with the SSL_VERIFY parameter because the value is interpreted by the Credentials class , using safe defaults and parsing boolean values permissibly from strings.

**Comment by jgarman** [comment]
OK I see that you're right - it's passed into the constructor for `Credentials` which does the magic I linked to. thanks.

**Pull request merged**